ckan/lib/webassets_tools.py

```diff
@@ -14,6 +15,8 @@
 logger = logging.getLogger(__name__)
 env = None
 
+yaml.warnings({u'YAMLLoadWarning': False})
```

Comment:
FYI I submitted a PR to webassets that got rid of the warning (https://github.com/miracle2k/webassets/pull/525) and it seems to be included in the latest version (2.0) so when we upgrade we can get rid of this.